Failed events alerts DQD (#1319)
snowplow/documentation: docs/data-product-studio/data-quality/failed-events/monitoring-failed-events/alerts/data-quality-alerts/managing-alerts/index.md

---
title: "Managing alerts"
sidebar_position: 2
---

> Edit, delete, or review existing failed events alerts.

Reviewer: ```suggestion
This page explains how to edit, delete, or review existing data quality alerts.
```

## View alerts

1. Navigate to **Data Quality** in the left sidebar
2. Click **Manage alerts** in the top-right corner
3. View all configured alerts with their destinations

![Manage alerts interface](images/dq_list_alerts.png)

## Edit an alert

1. Click the arrow next to the alert name
2. Modify destination, filters, or recipients
3. Click **Save** to update

## Delete an alert

1. Click the arrow next to the alert name
2. Click on the three dots button
3. Click **Delete**
4. Confirm deletion

> ### Multiple notifications

Reviewer: ```suggestion
## Multiple notifications
```

> Alerts trigger when new failed events match your filters. If you receive multiple notifications, check if:

Reviewer: ```suggestion
Alerts trigger when new failed events match your filters. If you unexpectedly receive multiple notifications, check if:
```
what does "multiple" mean here? multiple for the same event?

- Failed events are occurring frequently
- Filter criteria are too broad
- Multiple alerts have overlapping configurations

> ## Next steps

Reviewer: delete this section
